src/renderer/renderScene.ts

```diff
@@ -252,6 +252,19 @@ export function renderScene(
     if (isOutOfBounds) {
       context.globalAlpha = 0.2;
     }
+
+    if (
+      sceneState.remotePointerActivity &&
+      sceneState.remotePointerActivity[clientId] === "keydown"
+    ) {
+      context.beginPath();
+      context.arc(x, y, 15, 0, 2 * Math.PI, false);
+      context.lineWidth = 1;
+      context.stroke();
+      context.strokeStyle = strokeStyle;
+      context.closePath();
```

Comment:
I would recommend one circle with `lineWidth = 3` which will be white and the actual color inside with the `lineWidth = 1`. That way it will always be visible even if it's over a solid object with the same color!

Reply:
@lipis what should the fill color be to not clash with the cursor though? That's why I added a circle around.

Reply:
I thought you were already using the cursor's color for the circle.